scripts/test/procedures.py

```diff
@@ -1,8 +1,22 @@
 import shutil
 import subprocess
 from pathlib import Path
+import sys
 
-from scripts._job import Step, run_job
+
+def create_mypy_pkg_file():
+    pkg_path = [path for path in sys.path if path.endswith('site-packages')]
```

Comment:
A contextmanager would here also be nice, but I'm not sure whether that fits easily in the structure of having multiple steps.